lib/utils/jsonPathUtils.ts

```diff
@@ -48,7 +48,7 @@ export class JsonPathUtils {
    result: [ { value: 19, path: [ '$', 'details', 'information', 0, 'age' ] } ]
    */
   public static extractInputField(
-    obj: VerifiablePresentation | VerifiableCredential | PresentationDefinition,
+    obj: VerifiablePresentation | VerifiableCredential | VerifiableCredential[] | PresentationDefinition,
```

Comment:
Too many expected types.

Reply:
What do you suggest to do?

Reply:
param type should not be a union type of these but rather a more generic type that can accept any type of param, after all the extraction can work on any type of the param.
But this is just an improvement and is not a mandatory change.

Reply:
Changed to InputFieldType